akka-docs/src/main/paradox/stream/operators/Sink/collection.md

```diff
@@ -0,0 +1,26 @@
+# Sink.collection
+
+@scala[Collect all values emitted from the stream into a collection.]@java[Operator only available in the Scala API. The closest operator in the Java API is [`Sink.seq`](seq.html)].
```

Comment:
Perhaps this should only be in a `@@@div { .group-scala }` however I like that it offers a suggestion for Java users for the closest option, should they be looking.

Reply:
Thanks @helena for taking a look at this PR! My first preference would be for this document to not even appear in the Java operators list (at `stream/operators/index.md`), but the summary is generated and I'm not familiar enough with Paradox to know if this would be even possible? 

I pushed one more change where anything after this line is invisible in the Java version, with only the link back to the operator list staying visible. 

<img width="985" alt="image" src="https://user-images.githubusercontent.com/1792917/69267406-5d50d080-0bcd-11ea-840b-92130575a8b3.png">

Happy to hear any more feedback :)

Reply:
I believe it makes sense to show it in Java-mode with the note you suggested. People might be looking at Scala sample code and would this way be able to understand why they don't find it in the Java API.

Reply:
I think the top paragraph is pulled into the index/operator overview tables, does it really work using @scala/@java tags here? I think it needs to go in the description instead.

Reply:
Thanks @ennru and @johanandren for taking a look! 

I am not sure how much the average Java user is interested in details about the Scala API, so in this sense Helena's suggestion of hiding technical details in the Java section might be more sensible. 

I'm happy to put the Java description back if you feel that is a better way. And can for sure experiment with a `@@@note`.

Reply:
I like it like this! This needed a bit of fiddling to make the link work correctly from the index, that's a bit obscure so I took the liberty of pushing that to your branch with c045e564f5b0847ae99b5438766fb5836698e695